### PR TITLE
fix dependencies

### PR DESCRIPTION
**Pull request by SebastianElvis**
NOTE: commented out interchain account it's not used at the moment and the module panicks in e2e test.

### CODE REVIEW

File: demo/app/app.go [hunk 1]
```diff
@@ -564,21 +566,21 @@ func NewConsumerApp(
...
-	var noAuthzModule porttypes.IBCModule
-	icaControllerStack = icacontroller.NewIBCMiddleware(noAuthzModule, app.ICAControllerKeeper)
-	icaControllerStack = ibcfee.NewIBCMiddleware(icaControllerStack, app.IBCFeeKeeper)
-
-	// RecvPacket, message that originates from core IBC and goes down to app, the flow is:
-	// channel.RecvPacket -> fee.OnRecvPacket -> icaHost.OnRecvPacket
-	var icaHostStack porttypes.IBCModule
-	icaHostStack = icahost.NewIBCModule(app.ICAHostKeeper)
-	icaHostStack = ibcfee.NewIBCMiddleware(icaHostStack, app.IBCFeeKeeper)
+	// // Create Interchain Accounts Stack
```
Comment (maurolacy) [inline demo/app/app.go:569]: Double comments are not nice.

File: demo/go.mod [hunk 2]
```diff
@@ -7,11 +7,11 @@ toolchain go1.21.4
 require (
 	github.com/CosmWasm/wasmd v0.51.0
 	github.com/cosmos/cosmos-proto v1.0.0-beta.5 // indirect
-	github.com/cosmos/cosmos-sdk v0.50.6
+	github.com/cosmos/cosmos-sdk v0.50.7
 	github.com/cosmos/gogogateway v1.2.0 // indirect
 	github.com/cosmos/gogoproto v1.4.12
 	github.com/cosmos/iavl v1.1.2 // indirect
-	github.com/cosmos/ibc-go/v8 v8.0.0
+	github.com/cosmos/ibc-go/v8 v8.3.2
```
Comment (maurolacy) [inline demo/go.mod:14]: Perhaps this broke the ica stuff? 🤷🏼

File: tests/e2e/go.mod [hunk 3]
```diff
@@ -7,11 +7,11 @@ toolchain go1.21.4
 require (
 	github.com/CosmWasm/wasmd v0.51.0
 	github.com/cosmos/cosmos-proto v1.0.0-beta.5 // indirect
-	github.com/cosmos/cosmos-sdk v0.50.6
+	github.com/cosmos/cosmos-sdk v0.50.7
 	github.com/cosmos/gogogateway v1.2.0 // indirect
 	github.com/cosmos/gogoproto v1.4.12 // indirect
 	github.com/cosmos/iavl v1.1.2 // indirect
-	github.com/cosmos/ibc-go/v8 v8.0.0
+	github.com/cosmos/ibc-go/v8 v8.3.2
```
Comment (maurolacy) [inline tests/e2e/go.mod:14]: And here.

**Review by maurolacy — APPROVED** [review]
Looks good.

Perhaps the disabling of the ICA suff can be avoided? Or in any case, moved to a different PR?

**Comment by maurolacy** [comment]
E2E tests are still failing though.

**Comment by SebastianElvis** [comment]
Agreed, let's dedicate this PR for fixing dependencies. Will create another PR for fixing CI / removing ICA after this one

File: demo/go.mod (hunk 2 above)
Comment (SebastianElvis) [reply demo/go.mod:14]: I tried v8.0.0 and it does not make any difference, weird

Comment (maurolacy) [reply demo/go.mod:14]: Likely v8.3.x is being pulled through the dependency tree.

Comment (maurolacy) [reply demo/go.mod:14]: That, or the change comes from somewhere else (`cosmos-sdk`).

**Pull request merged**